db/create_tables.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE poll(
+    id NUMERIC,

```

Comment:
PostgreSQL の numeric 型は正確さが求められる数値計算で利用する型のようです。
https://www.postgresql.jp/document/12/html/datatype-numeric.html

単なる primary key として利用するだけなら integer/bigint で十分そうです。
また、serial/bigserial を使えば、新しいレコードの挿入時に id が自動で設定されるようになるようなので、これを利用しても良いかなと思います。

```suggestion
    id bigserial,
```